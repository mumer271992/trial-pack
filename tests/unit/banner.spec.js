/* eslint-disable no-undef */
import { mount } from '@vue/test-utils';
import BannerComponent from '../../src/components/Banner.vue';

describe('Testing banner component', () => {
  test('renders correctly', () => {
    const wrapper = mount(BannerComponent);
    expect(wrapper.element).toMatchSnapshot();
  });
});
