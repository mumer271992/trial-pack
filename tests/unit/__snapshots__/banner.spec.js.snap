// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing banner component renders correctly 1`] = `
<section
  id="banner_section"
>
  <div
    class="container f-h"
  >
    <div
      class="banner_section_content c-row align-center f-h"
    >
      <div
        class="c-col-lg-3 c-col-md-4 c-col-sm-6 c-col-xs-12 t-c"
      >
        <h1
          class="t-c"
        >
          Discover our skin friendliness
        </h1>
         
        <button
          class="primary large"
        >
          Now choose your size
        </button>
      </div>
    </div>
  </div>
</section>
`;
